### ISSUE TITLE
TreeView & W3C standards

### ISSUE THREAD

**Issue by Eevolee** [Jahia/moonstone#1070]
### Keyboard interactions:
- Home & End
- `*` key (Optional): Expands all siblings that are at the same level as the current node.
### Attribute changes:
- Elements without children shouldn't have aria-expanded attribute
- ~~Children containers (=nested lists) should have role="group"~~ (nested lists being `li` elements instead of `ul` or `ol`, adding role="group" causes accessibility issues. Nested lists should contain a `ul` or `ol` element to set the role to)

**Comment by ffffffelix** [comment]
- We do not implement Home and End and a shortcut to expand siblings
- Attribute changes: communicate with qa team to know if it can break some tests
